### ISSUE TITLE
Y/N detection (0.3.DL)

### ISSUE THREAD

**Issue by NotNutter** [oscar-broman/SKY#33]
Hello,
I am trying the SKY binaries on Debian 10 for my 0.3.DL server as I plan to migrate. However, I have found that KEY_NO is not detected as a passenger with SKY. I have also found that the client's health bar does not update when they lose/gain health. The issues are non-existent on older versions of SKY.

The detection of KEY_NO is pretty important for my players so I do not mind testing hotfixes on the server I am using to test the migration.

**Comment by ADRFranklin** [comment]
The latest version should not have any issues with healthbars or KEY_NO.

Though I do not use the 0.3DL version myself, Crayder sure does and he has confirmed that this all works. So I'm going to need all the information you can give.

**Comment by NotNutter** [comment]
I am using Debian 10 and using the binaries provided with the lastest release of SKY. 

[release-linux.tar.gz](https://github.com/oscar-broman/SKY/files/3563792/release-linux.tar.gz)

I have tried my gamemode which uses weapon-config and then I tested a bare script without weapon-config but with SKY where KEY_NO was not detected as a passenger in a vehicle. 

I then tried the bare script without SKY and then I see KEY_NO/KEY_YES being detected successfully when I was a passenger in a vehicle.

**Comment by NotNutter** [comment]
I have now been able to recreate this on the windows server using the version on the release page too. I will upload the small testing script I used to re-produce the issues I mention above.

**Comment by ADRFranklin** [comment]
Okay, great I haven't been able to re-produce this myself right now, so that test script could be what I need to fix it.

**Comment by NotNutter** [comment]
This is the script I am using to test the isssue. https://gist.github.com/NotNutter/972bc1cdb08eed536ada1487021cf5b3

I have done some further testing to see why the weapon-config healthbar textdraws weren't updating. I found that the healthbar textdraws would update while using an older version of SKY whereas they did not update when I loaded the latest released version of SKY.

I have deployed a 0.3.DL server with the above script at 51.68.212.165:7777.
